config/default.yml

```diff
@@ -3,182 +3,68 @@ require:
   - rubocop-performance
 
 AllCops:
-  DisabledByDefault: true
+  EnabledByDefault: true
```

Comment:
I think that this repository should not declare either `DisabledByDefault` nor `EnabledByDefault` and leave that to the downstream inheritor of this configuration.

That way this ruleset can be completely verbose of "these are the baseline rules", and inheriting projects can choose to further inherit all of the Rubocop rules however they want.

I'm not 100% sure on this, I think it will have the same effect in this repository, but will effect downstream repositories.

Reply:
I suppose it'll happen either way, but (relevant to our chat earlier), maybe setting `DisabledByDefault` was so that whichever team first made this gem would be an authority on Ruby at GitHub, and other downstream consumers diverged for their own code style requirements. Is _that_ maybe why it's not 100% compatible with the styleguide now?

Reply:
Oops, I shouldn't have written "this repository"; I specifically meant `config/default.yml` shouldn't define neither `DisabledByDefault` nor `EnabledByDefault`. 

The rules for this repository, `.rubocop.yml` should probably have `EnabledByDefault`.

Reply:
Riiiight. I see. Makes sense, I think?

Reply:
So, to recap:

- You'd like `.rubocop.yml` in _this repository_ (the RuboCop config for the code in this gem) to set `EnabledByDefault`.
- You would like `config/default.yml` (which all of the downstream consumers of this gem inherit their rules from) to not set either `EnabledbyDefault` or `DisabledByDefault`? That means that it's just using the default RuboCop behaviour, which is probably a safe bet.

I'm not sure what benefit this first thing has, since the RuboCop defaults are similar enough?